standard/standard/clause_0_front_material.adoc

```diff
@@ -1,89 +1,98 @@
+JSON-Schema:2019-9 encoding specification. The key standardization requirements
+specify that concrete JSON-encoded GeoPose data objects must conform to the
+corresponding JSON-Schema definition. The individual elements identified in the
+encoding specification are composed of SDUs, tying the specifications back to the
+Logical Model.
+
+The GeoPose 1.0 Standard makes no assumptions about the interpretation of external
+specifications, for example, of reference frames. Nor does it assume or constrain
+services or interfaces providing conversion between GeoPoses of difference types or
+relying on different external reference frame definitions.
+
+== Preface
 
 [NOTE]
 ====
-This is a draft implementation specification for GeoPose 1.0. 
-Attention is drawn to the possibility that some of the elements of this document may be the subject of patent rights. The Open Geospatial Consortium shall not be held responsible for identifying any or all such patent rights.
-
-Recipients of this document are requested to submit, with their comments, notification of any relevant patent claims or other intellectual property rights of which they may be aware that might be infringed by any implementation of the standard set forth in this document, and to provide supporting documentation.
+This is a draft implementation specification for GeoPose 1.0.
+Attention is drawn to the possibility that some of the elements of this document may
+be the subject of patent rights. The Open Geospatial Consortium shall not be held
+responsible for identifying any or all such patent rights.
+
+Recipients of this document are requested to submit, with their comments,
+notification of any relevant patent claims or other intellectual property rights of
+which they may be aware that might be infringed by any implementation of the standard
+set forth in this document, and to provide supporting documentation.
 ====
-[big]*iv.    Submitting organizations*
-
-The following organizations submitted this Document to the Open Geospatial Consortium (OGC):
-
-[NOTE]
-The organizations, submitters, participants, and acknowledgements are open to changes and additions - just email a chair or editor.
-
-Organization name(s)
-
-This Document was submitted to the Open Geospatial Consortium (OGC) by the members of the GeoPose Standards Working Group of the OGC. Amongst others, this comprises the following organizations:
-
-* Norkart AS
-* Open Site Plan
-* Open AR Cloud Association
-* Ordnance Survey
-* School of Management and Engineering Vaud, HES-SO University of Applied Sciences and Arts Western Switzerland
 
+== Submitters
 
+All questions regarding this submission should be directed to the editors or the
+submitters:
 
-[big]*v.     Submitters*
-
-All questions regarding this submission should be directed to the editors or the submitters:
-
-[#submission_contact_points,reftext='{table-caption} {counter:table-num}']
 .Submission Contact Points
-[width="100%",cols="5,10",options="header"]
```

Comment:
Remove the `cols=...` as well. This is a semantic section.

Reply:
Removed.